app/static/js/user_schedule.js

```diff
@@ -0,0 +1,213 @@
+        if(project.id === prev_project.id){
+            APP.toast("Project already exists!");
+            return;
+        }
+    }
+    // Add project to list
+    table_data.data.projects.push(project);
+
+    // Add zeros to array, this is the new column
+    table_data.data.schedule.forEach(element => {
+        element.push(0);
+    });
+
+    // Add header (project name)
+    table_data.project_names.push(project.name);
+
+    // Render table with new data
+    table_data.hot.render();
+
+    // Make sure the new column gets the numeric renderer
+    table_data.hot.updateSettings({
+        columns: function(index){
+            return {
+                type: 'numeric',
+                allowEmpty: false
+            }
+        }
+    });
+}
+
+
+APP.register_module(async function(){
+    APP.team = await $.post('/api/me/team').promise();
+    table_data.year = moment().year();
+    table_data.container = $('#schedule-table');
+
+    table_data.hot = new Handsontable(table_data.container[0], {
+        data: [[]],
+        rowHeaderWidth: 160,
+        afterGetRowHeader: (col, TH) => {
+            TH.className = 'left';
+        },
+        afterChange: function(changes, source) {
+            console.log(source, changes);
+            if(source === 'edit' || source === 'Autofill.fill'){
+                if(changes.length === 1){
+                    send_update(changes[0]);
+                }
+                else{
+                    send_updates(changes);
+                }
+            }
+        },
+        cells: function (row, col, prop) {
+            var cellProperties = {}
+            return cellProperties;
+        },
+        columns: function(index){
+            return {
+                type: 'numeric',
+                allowEmpty: false
+            }
+        }
+    });
+
+    $("#prev-button").on('click', (event) => {
+        event.preventDefault();
+        table_data.year--;
+        render_table();
+    });
+
+    $("#next-button").on('click', (event) => {
+        event.preventDefault();
+        table_data.year++;
+        render_table();
+    });
+
+    render_table();
+    
+    $("#project-name-add").focus(async function(){
+        let options = await $.post(`/api/team/${APP.team.id}/projects`);
+        console.log(options);
+        options = options.map(element => {
+            return {
+                label: element.name,
+                value: element
+            }
+        });
+        console.log(options);
+        $("#project-name-add").autocomplete({
+            source: options,
+            focus: function(event, ui) {
+                $("#project-name-add").val( ui.item.label );
+                return false;
+            },
+            select: function(event, ui){
+                event.preventDefault();
+                $("#project-name-add").val( ui.item.label );
+                addProject(ui.item.value);
+                return false;
+            }
+          });
+    });
+
+});
```

Comment:
ASCII character 0x0A plz
```suggestion
});

```